pkg/odo/cli/cli.go

```diff
@@ -158,6 +158,11 @@ func odoRootCmd(name, fullName string) *cobra.Command {
 	_ = pflag.CommandLine.MarkHidden("log_dir")
 	_ = pflag.CommandLine.MarkHidden("logtostderr")
```

Comment:
shouldn't this one be removed as well?

Reply:
It still seems to be present here @dev-gaur https://github.com/openshift/odo/pull/3573/files#diff-6bde9f69cdf69930cd6e07b7e02961a5R46

Reply:
My bad, these are different

Reply:
It is still present https://github.com/openshift/odo/pull/3573/files#diff-6bde9f69cdf69930cd6e07b7e02961a5R51